tests/jest/workspace/DynamicOverlayContainer.test.jsx

```diff
@@ -0,0 +1,186 @@
+import React from 'react';
```

Comment:
You can remove these net-new test files as well, they should be present in the /tests/jest/shared directory.

Reply:
Updated.